server/tests/e2e/reports.e2e-spec.ts

```diff
@@ -0,0 +1,280 @@
+// noinspection DuplicatedCode
+
+import { PrismaService } from '@modules/repo/prisma.service';
+import { post } from '../util/request-handlers.util';
+import { AuthService } from '@modules/auth/auth.service';
+import { ReportType, ReportCategory } from '@common/enums/report.enum';
+import { ReportDto } from '@common/dto/report/report.dto';
+
+describe('Reports', () => {
+    let adminUser,
+        adminUserToken,
+        adminGameUserToken,
+        adminUser2,
+        modUser,
+        modUser2,
+        nonAdminUser,
+        nonAdminAccessToken,
+        modUserToken,
+        user1,
+        user2;
+
+    /*
+    let adminAccessToken, adminGameAccessToken;
+    const testUser = {
+        id: 1,
+        steamID: '00000000000000001',
+        roles: user.Role.VERIFIED,
+        bans: 0
+    };
+    const testAdmin = {
+        id: 2,
+        steamID: '00000000000000002',
+        roles: user.Role.ADMIN,
+        bans: 0
+    };
+    const testAdminGame = {
+        id: 3,
+        steamID: '00000000000000003',
+        roles: user.Role.ADMIN,
+        bans: 0
+    };
+
+    const testMap = {
+        id: 1,
+        name: 'test_map',
+        type: map.MAP_TYPE.UNKNOWN,
+        statusFlag: map.STATUS.APPROVED,
+        submitterID: testUser.id,
+        info: {
+            description: 'newmap_5',
+            numTracks: 1,
+            creationDate: new Date()
+        },
+        tracks: [
+            {
+                trackNum: 0,
+                numZones: 1,
+                isLinear: false,
+                difficulty: 5
+            }
+        ],
+        credits: {
+            id: 1,
+            type: map.CreditType.AUTHOR,
+            userID: testUser.id
+        }
+    };
+	*/
```

Comment:
Lets remove all this commented out stuff, it's just tests from the old API. Now we're ported/rewritten them we don't need the comments (applies to this whole file).